app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -314,11 +315,25 @@ class SuggestedEditsTasksFragment : Fragment() {
         addDescriptionsTask.primaryAction = getString(R.string.suggested_edits_task_action_text_add)
         addDescriptionsTask.secondaryAction = getString(R.string.suggested_edits_task_action_text_translate)
 
+        imageRecommendationsTask = SuggestedEditsTask()
+        imageRecommendationsTask.title = getString(R.string.suggested_edits_image_recommendations_task_title)
+        imageRecommendationsTask.description = getString(R.string.suggested_edits_image_recommendations_task_detail)
+        imageRecommendationsTask.imageDrawable = R.drawable.ic_add_image
+        imageRecommendationsTask.primaryAction = getString(R.string.suggested_edits_task_action_text_add)
+        imageRecommendationsTask.new = !Prefs.suggestedEditsImageRecsOnboardingShown
+
         if (DescriptionEditUtil.wikiUsesLocalDescriptions(WikipediaApp.instance.wikiSite.languageCode) && viewModel.blockMessageWikipedia.isNullOrEmpty() ||
             !DescriptionEditUtil.wikiUsesLocalDescriptions(WikipediaApp.instance.wikiSite.languageCode) && viewModel.blockMessageWikidata.isNullOrEmpty()) {
             displayedTasks.add(addDescriptionsTask)
         }
 
+        if (ReleaseUtil.isPreBetaRelease &&
```

Comment:
Not sure but are we going to release this feature to the production in the upcoming release?

Reply:
I've changed the feature flag to PreProd for now.